indico/modules/events/papers/templates/reviewing/timeline.html

```diff
@@ -79,16 +79,77 @@ <h3 class="f-self-stretch">
-        {% trans -%}
-            Proposed to {{ action }}correct{{ endaction }}
-        {%- endtrans %}
+        {%- trans %}Proposed to {{ action }}correct{{ endaction }}{% endtrans -%}
+    {% endif %}
+{% endmacro %}
+
+{% macro render_paper_judgment(revision) %}
+    {% if revision.state.name != 'submitted' %}
+        <div class="i-timeline-item">
+            {{ render_user_avatar(revision.judge) }}
+            <div class="i-timeline-item-box header-indicator-left">
+                <div class="i-box-header flexrow">
+                    <div class="f-self-stretch">
+                        {{ _render_paper_judgment_action(revision) }}
+                        <time datetime="{{ revision.judgment_dt.isoformat() }}">
+                            {{- revision.judgment_dt|format_human_date -}}
+                        </time>
+                    </div>
+                    {% if revision.paper.event_new.can_manage(session.user) %}
+                        <div>
+                            <a class="i-link icon-remove"
+                               title="{% trans %}Reset judgement{% endtrans %}"
+                               data-href="{{ url_for('.reset_paper_state', revision.paper) }}"
+                               data-method="POST"
+                               data-update="#reviewing-page"
+                               data-replace-update
+                               data-title="{% trans %}Reset judgment{% endtrans %}"
+                               data-confirm="{% trans %}Do you really want to reset the judgment? This operation is irreversible.{% endtrans %}"></a>
+                        </div>
+                    {% endif %}
+                </div>
+                <div class="i-box-content">
+                    {% if revision.state.name == 'accepted' %}
+                        {%- trans %}The paper was accepted.{% endtrans -%}
+                    {% elif revision.state.name == 'rejected' %}
+                        {%- trans %}The paper was rejected.{% endtrans -%}
+                    {% elif revision.state.name == 'to_be_corrected' %}
+                        {%- trans %}The paper requires changes{% endtrans -%}
+                    {% endif %}
+                    {% if revision.judgment_comment %}
+                        <div class="titled-rule">
+                            {%- trans %}Comment{% endtrans -%}
+                        </div>
+                        {{ revision.judgment_comment }}
+                    {% endif %}
+                </div>
+            </div>
+        </div>
+    {% endif %}
+{% endmacro %}
+
+{% macro _render_paper_judgment_action(revision) %}
+    {% set action = revision.state.name %}
+    {% if revision.paper.can_judge(session.user) %}
+        {% set judge_name = revision.judge.name %}
+        {% if action == 'accepted' %}
+            {%- trans name=judge_name %}<strong>{{ name }}</strong> accepted this paper{% endtrans -%}
```

Comment:
Why not use `judge_name` directly instead of having `name=judge_name` all the time?